azure-pipelines.yml

```diff
@@ -36,7 +41,6 @@ stages:
         Dockerfile: '$(Build.SourcesDirectory)/Dockerfile'
         containerRegistry: $(SERVICE_CONNECTION)
         tags: |
-          $(tag)
-          alpha
-          $(majorMinorPatchVersion)
-          
+          $(primaryTag)
```

Comment:
in primary tag we shall add the 
develop in the build otherwise the develop builds will override main build

i suggest as developer tag : 
    develop-latest

Reply:
Rest is a good idee